tests/integration/resin.spec.coffee

```diff
@@ -29,28 +29,30 @@ describe 'Resin SDK', ->
 	it 'should expose a resin-pine instance', ->
 		m.chai.expect(resin.pine).to.exist
 
-	it 'should expose a resin-token instance', ->
-		m.chai.expect(resin.token).to.exist
-
 	it 'should expose an resin-errors instance', ->
 		m.chai.expect(resin.errors).to.exist
 
 	describe 'interception Hooks', ->
 
-		givenLoggedInUser()
-
 		beforeEach ->
 			resin.interceptors = []
 
 		afterEach ->
 			resin.interceptors = []
 
-		it "should update if the array is set directly (not only if it's mutated)", ->
-			interceptor = request: m.sinon.mock().returnsArg(0)
-			resin.interceptors = [ interceptor ]
+		givenLoggedInUser()
+
+		getRequestInfo = (fn) ->
```

Comment:
I don't understand this method name. How it this getting request info? Maybe `ignoreWhoamiCalls` or something instead?